src/elements/Banner/Banner.tsx

```diff
@@ -61,19 +57,12 @@ export class Banner extends React.Component<BannerProps> {
   render() {
     if (this.state.dismissed) return null
     const showCloseButton = this.props.dismissable
-    const textAlignment = showCloseButton ? "left" : "center"
-    const paddingRight = showCloseButton ? 6 : 2
 
     return (
-      <Wrapper
-        bg={this.props.backgroundColor}
-        color="white100"
-        p={1}
-        position="relative"
-        pr={paddingRight}
-        textAlign={textAlignment}
-      >
-        <Sans size="2">{this.props.message}</Sans>
+      <Wrapper bg={this.props.backgroundColor} color="white100" p={1}>
+        <TextWrapper>
```

Comment:
This `TextWrapper` component can completely go away and width / centering can be passed into the wrapper